appengine/storage/app.js

```diff
@@ -35,8 +36,8 @@ app.set('view engine', 'pug');
 // [START config]
 // Multer is required to process file uploads and make them available via
 // req.files.
-const multer = require('multer')({
-  inMemory: true,
+const multer = Multer({
+  storage: Multer.MemoryStorage,
```

Comment:
From multer's readme it seems like this should be `storage: Multer.memoryStorage()` right?